src/vote/vote.objecttype.dto.ts

```diff
@@ -0,0 +1,37 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql';
+
+@ObjectType('Vote')
+export class Vote {
+  @Field(() => Int)
```

Comment:
논리상 id 에 해당한다면,
graphql scalar id 타입을 사용해보면 어떨까요?

```suggestion
  @Field(() => ID)
```

- https://graphql.org/learn/schema/#scalar-types